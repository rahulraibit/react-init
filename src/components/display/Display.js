import React from 'react';

class Display extends React.Component {
    render() {
        return (
            <div>
                {'Hi I am Display component'}
            </div>
        );
    }
}

export default Display;