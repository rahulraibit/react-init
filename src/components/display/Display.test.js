import React from 'react';
import Display from './Display'
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

//snapshot testing
test('Display the content', () => {
    const component = renderer.create(
        <Display />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})

// Dom testing
it('renders welcome message', () => {
    const { getByText } = render(<Display />);
    expect(getByText('Hi I am Display component')).toBeInTheDocument();
  });