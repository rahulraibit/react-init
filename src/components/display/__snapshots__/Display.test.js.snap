// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display the content 1`] = `
<div>
  Hi I am Display component
</div>
`;
